build/gulp/tasks/test-vulns.ts

```diff
@@ -0,0 +1,65 @@
+import * as fs from 'fs'
+import { task } from 'gulp'
+import * as path from 'path'
+import debug from 'debug'
+
+import config from '../../../config'
+import sh from '../sh'
+
+const { paths } = config
+
+const SCAN_RESULTS_DIR_NAME = '.vuln-scans'
+
+const log = message => debug.log(message)
+log.success = message => debug.log(`✔ ${message}`)
+
+const ensureDirExists = path => {
+  if (!fs.existsSync(path)) {
+    sh(`mkdir -p ${path}`)
```

Comment:
`fs.mkdirSync(path, { recursive: true })`?

Reply:
for that we need Node LTS v10 (and this is, actually, the step we **should** make). Agreed to defer it to the follow-up PR, to be absolutely sure that all the necessary accompanying adjustments to the code will be made